Makefile

```diff
@@ -34,7 +34,7 @@ help:
 
 # Install system packages (for use with containers)
 deps-ubuntu:
-	apt-get install imagemagick
+	apt-get -y install imagemagick
```

Comment:
The old code also used `--no-install-recommends` which I think is good to keep the size of the installation as small as possible, especially for Docker images.

Reply:
In this case, the recommendations are all valuable for the processor: 
- `libmagickcore-6.q16-3-extra` (more converters)
- `netpbm`
- `ghostscript`